src/main/java/org/ohdsi/circe/check/checkers/UnusedConceptsCheck.java

```diff
@@ -51,14 +52,19 @@ protected WarningReporter getReporter(WarningSeverity severity, List<Warning> wa
     @Override
     public void check(CohortExpression expression, WarningReporter reporter) {
 
-        for(final ConceptSet conceptSet : expression.conceptSets) {
+        List<Criteria> additionalCriterion;
```

Comment:
Just a nitpick, but:  if it is a collection, then it's many things, so wouldn't it be _additionalCriteria_?  The singular would be criterion.  If it was in a for loop it would be `for (criterion in criteria) { }`

For simplicity, we always refer to these things as criteria becaues even in the 'single instance' case, the object is a collection multiple criteria (the age criterion, the gender criterion, etc).

I'd just like to avoid confusing the terminology.